app/services/ping.rb

```diff
@@ -12,13 +12,23 @@ def ping
     end
 
     def statuses
+      plugins = []
+      Foreman::Plugin.all.each do |plugin|
+        plugins << {
+          id: plugin.id,
+          name: plugin.name,
+          version: plugin.version,
+          author: plugin.author,
+          description: plugin.description,
+        }
+      end
```

Comment:
Shouldn't this work as well?
```suggestion
      plugins = Foreman::Plugin.all.map do |plugin|
        {
          id: plugin.id,
          name: plugin.name,
          version: plugin.version,
          author: plugin.author,
          description: plugin.description,
        }
      end
```

Reply:
see, yellin' at ma Ruby!